### DISCUSSION TITLE
Request for Edge-to-Edge Layout Implementation

### DISCUSSION THREAD

**Discussion by Amorphous404** [community/community#67954] [Mobile] [answered]
Hi,

I'm a dedicated user of GitHub app and I truly appreciate the functionality it offers. I wanted to reach out with a suggestion that I believe could enhance the user experience of the app even further.

In recent updates of mobile operating systems, the concept of edge-to-edge design has gained significant popularity. This design approach not only provides a sleek and modern visual look but also optimizes the use of screen real estate, making it a more immersive experience for users.

Example:
![Edge to Edge Layout(2).jpg](https://github.com/community/community/assets/96917972/fbfedaf2-7483-40eb-829f-3c57c77113d7)

I kindly request that you consider implementing it. Doing so would not only improve the app's aesthetics but also make the most of the available screen space, ultimately enhancing the overall usability.

To assist you in this endeavor, I've come across a resource that might be helpful. where you'll find valuable insights related to edge-to-edge design principles and best practices.

Display content edge-to-edge in your app: 
https://developer.android.com/develop/ui/views/layout/edge-to-edge#lay-out-in-full-screen

I understand that implementing such a change might require time and effort, but I believe it would be a valuable addition that many users, including myself, would greatly appreciate.

Thank you for your continuous efforts in making the GitHub a fantastic app. I look forward to any improvements you make and appreciate your dedication to enhancing the user experience.

Thanks,

**Comment by stevepopovich** [comment, marked as answer]
Hi @Amorphous404 thanks for the feedback. I passed it onto the team for consideration!
